src/_posts/languages/php/2000-01-01-laravel.md

```diff
@@ -218,23 +218,46 @@ of your application:
 queues: php artisan queue:work --queue=high,default
 ```
 
-The arguments of the `--queue` option must be customized to suit your needs. After deploying your
-application, you must scale the newly created type of containers `queues` at least to 1.
+The arguments of the `--queue` option must be customized to suit your needs.
+After deploying your application, you must scale the newly created type of
+containers `queues` at least to 1.
 
 ## Laravel Tasks Scheduler
 
-The Laravel PHP framework includes a command scheduler for asynchronous tasks. However, Laravel's
-[documentation](https://laravel.com/docs/8.x/scheduling#introduction) states you need to add a cron
-entry which is not possible on Scalingo.
+The Laravel PHP framework includes a command scheduler for asynchronous tasks.
+However, Laravel's [documentation](https://laravel.com/docs/9.x/scheduling#running-the-scheduler)
+states you need to add a cron entry which is not possible on Scalingo.
 
-{% note %}
-The cron feature is now available upon request. Refer to [this page]({% post_url platform/app/task-scheduling/2000-01-01-scalingo-scheduler %}) for more information.
-{% endnote %}
+From this point, there are two possibilies, depending on the frequency of the
+scheduled tasks.
 
-This documentation page guides you to let your application use the Laravel command scheduler on
-Scalingo. The following command must be added to your app. It calls the Laravel scheduler every
-minutes. It requires you to start [another process]({% post_url platform/app/2000-01-01-procfile %})
-in your application.
+### The Tasks Run on a Frequency Lower Than Once Every 10 Minutes
+
+For these cases, we highly recommend to use the [Scalingo Scheduler]({% post_url platform/app/task-scheduling/2000-01-01-scalingo-scheduler %}).
+
+All you have to do is create a `cron.json` file at the root of your application
+with a content similar to this:
```

Comment:
We usually avoid wording which suggests that something is simple or obvious. That's why we don't use the word "just" or "simply" in the doc. And I have the feeling that the wording "All you have to do" is that kind of construct. But may be just me being picky, as you wish :) 

```suggestion
Create a `cron.json` file at the root of your application with a content similar to this:
```